widgets/EmailBySendgrid-postaction.json

```diff
@@ -0,0 +1,51 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "configuration-groups": [
+    {
+      "label": "Email Properties",
+      "properties": [
+        {
+          "widget-type": "select",
```

Comment:
for newer plugins, we've been using radio-group with 'inline' attribute for properties that have 3 possible values.